officehours.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Office Hours
+sidebar_link: true
+sidebar_sort_order: 250
+---
+
+Boston Python holds a weekly office hour, Mondays at noon.
+
+This is an unstructured time to drop in to ask questions or just chat. There is no set topic, anything goes: Python, Boston Python, specialized technical topics, adjacent technologies, it's all good.  For now, this is an experiment. We may change the structure, pace, or timing as we see what happens.
+
+The office hour happens on [Zoom](https://edx-org.zoom.us/j/85746000546?pwd=N3RyR2VsbGFMTFdPUUUva3VhTlFHdz09), sponsored by [edX](https://edx.org).
+
+You do not have to sign up ahead of time, just drop in.  See you there!
```

Comment:
```suggestion
You do not have to sign up ahead of time; just [drop in on Zoom](https://edx-org.zoom.us/j/85746000546?pwd=N3RyR2VsbGFMTFdPUUUva3VhTlFHdz09).  See you there!
```

Reply:
@nedbat:

> I could leave the Zoom link out, and tell people to find the meetup.com event page instead. Meetup only shows the link to members, I think?

Don't know; that would add some friction to people joining. I don't feel strongly; just a thought.